server/services/world/party/party.controller.ts

```diff
@@ -0,0 +1,25 @@
+import {Controller}      from "@nestjs/common";
+import {PartyGateway}    from "./party.gateway";
+import {EventPattern}    from "@nestjs/microservices";
+import {WORLD_PREFIX}    from "../world.prefix";
+import {MadePartyLeader} from "../../party/actions";
+import {PartyClient}     from "../../party/client/party-client.service";
+
+@Controller()
+export class PartyController {
+
+    constructor(
+        private gateway: PartyGateway,
+        private client: PartyClient
+    ) {
+
+    }
+
+    @EventPattern(WORLD_PREFIX + MadePartyLeader.event)
+    async onMadePartyLeader(data: MadePartyLeader) {
```

Comment:
Reacting to the party leader change here makes it so all instances of the world servers can report the change to their players.